client/stylesheets/_modal.scss

```diff
@@ -1181,10 +1181,62 @@ div[class="sp-replacer sp-light full-spectrum sp-active"] {
 	END END END END END END
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+	Profile Active Modal
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 
+.modProfileActiveBtn {
+	margin: 20px 0 !important;
+	color: #fff !important;
+}
 
+.modProfileActiveBtn button {
+	padding: 15px !important;
+	width: 75% !important;
+	color: #fff !important;
+	background-color: #f8504b !important;
+	border-radius: 4px;
+}
 
+.modProfileActiveBtn button:hover {
```

Comment:
Merge rule `.modProfileActiveBtn button:hover` with rule on line 1188<br>Selector `modProfileActiveBtn` should be written in lowercase with hyphens